src/library_webgpu.js

```diff
@@ -1853,8 +1853,10 @@ var LibraryWebGPU = {
       // TODO(kainino0x): Somehow inject a validation error?
       return 0;
     }
-
-    var data = _memalign(16, mapped.byteLength);
+    const mappedAlignment = {{{ to64(16) }}};
+    const mappedSize = {{{ to64('mapped.byteLength') }}};
+    var data = _memalign(mappedAlignment, mappedSize);
+    {{{ from64('data') }}}
```

Comment:
Instead of this can you add `memalign` to `create_pointer_conversion_wrappers`

Reply:
Ah, didn't know that such a thing existed and wondered how malloc worked. Happy to remove these gymnastics in favor of doing that.

Reply:
Done.